diff --git a/node_modules/mammoth/lib/xml/xmldom.js b/node_modules/mammoth/lib/xml/xmldom.js
--- a/node_modules/mammoth/lib/xml/xmldom.js
+++ b/node_modules/mammoth/lib/xml/xmldom.js
@@ -5,12 +5,12 @@ function parseFromString(string) {
     var error = null;
 
     var domParser = new xmldom.DOMParser({
-        errorHandler: function(level, message) {
+        onError: function(level, message) {
             error = {level: level, message: message};
         }
     });
 
-    var document = domParser.parseFromString(string);
+    var document = domParser.parseFromString(string, "application/xml");
 
     if (error === null) {
         return document;
